src/react/boosts/payments/BoostPayFormEth.tsx

```diff
@@ -60,14 +73,18 @@ export class BoostPayFormEth extends React.Component<BoostPayFormEthProps, Boost
     return (
       <BoostPayFormWrapper>
         <form>
-          {/*<BoostPayFormContain>
-            <BoostPayFormTitle>Your name (optional)</BoostPayFormTitle>
-            <TextInput name="userName" />
-            <BoostPayFormTitle>Include a comment (optional)</BoostPayFormTitle>
-            <TextInput name="userComment" />
-            <Checkbox onClick={this.onClick} checked={this.state.checked} id={""} />
-            <CheckboxLabel>Email me about progress of this Boost</CheckboxLabel>
-          </BoostPayFormContain>*/}
+          <BoostUserInfoForm>
+            <label>Email (optional)</label>
+            <BoostInput
+              valid={this.state.validEmail}
+              id="email"
+              name="email"
+              type="email"
+              maxLength={254}
+              onChange={() => this.handleOnChange(event)}
```

Comment:
definitely ok for now, but eventually we should do that fancy thing where it only validates once the user stops typing. I saw a really slick way of doing this using observables a little while back, I'll see if I can find it again (it honestly might've been a Consensys dev session)

Reply:
Agreed I'm not a fan of onChange validation but it works for now